speechmatics/client.py

```diff
@@ -519,6 +519,33 @@ def run_synchronously(self, *args, timeout=None, **kwargs):
         # pylint: disable=no-value-for-parameter
         asyncio.run(asyncio.wait_for(self.run(*args, **kwargs), timeout=timeout))
 
+    async def send_message(self, msg):
+        """
+        Sends a message to the server.
+        """
+        if self.session_running:
+            assert self.websocket, "Websocket hasn't been instantiated"
+            try:
+                LOGGER.info(f"Sending msg={msg}")
+                await self.websocket.send(json.dumps(msg))
```

Comment:
you can use`websocker.send_json()` instead

Reply:
It adds a bit of error checking and helpful messages back to the user + it asserts that websocket is not None, thought it would be nicer to have it?

Reply:
My bad, I was thinking of fastapi.websocket but that's not what we're using here

Reply:
you're implementing our own version of `send_json()` which what this function should be called?

Reply:
yes